packages/ui-core/shared/src/lib/dashboard/widgets/charts/employee-doughnut-chart-widget.component.html

```
@@ -0,0 +1,10 @@
+<ga-employee-chart-card
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)